.github/workflows/awie.yml

```
@@ -24,6 +24,19 @@ env:
   module: awie
 
 jobs:
+  scan:
+    uses: ./.github/workflows/veracode-pipeline-scan.yml
+    with:
+      module_directory: centreon-awie
+      module_name: centreon-awie
+      cache_key: ${{ github.sha }}-${{ github.run_id }}
+      target_branch_name: ${{ github.base_ref || 'develop' }}
```

```suggestion
      target_branch_name: ${{ github.base_ref || github.head_ref_name }}
```
To replace before merging